go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.19.0
 	github.com/mitchellh/hashstructure v1.1.0
 	github.com/onsi/gomega v1.22.1
-	github.com/xanzy/go-gitlab v0.73.1
+	github.com/xanzy/go-gitlab v0.74.0
```

Comment:
`go-gitlab@v0.74.0` will be integrated with https://github.com/gitlabhq/terraform-provider-gitlab/pull/1308. Thus, another rebase may be required ;)

Reply:
Okay, I'll keep an eye on it and do another rebase in case #1308 gets merged before this :)

Reply:
It's merged btw :)